demos/textfield.html

```diff
@@ -294,6 +375,55 @@
       })();
     </script>
     <script>
+      setTimeout(function() {
+        var tfLeadingEl = document.getElementById('tf-box-leading-example');
+        var tfLeading = new mdc.textfield.MDCTextfield(tfLeadingEl);
+        var wrapperLeading = document.getElementById('demo-tf-box-leading-wrapper');
+
+        var tfTrailingEl = document.getElementById('tf-box-trailing-example');
+        var tfTrailing = new mdc.textfield.MDCTextfield(tfTrailingEl);
+        var wrapperTrailing = document.getElementById('demo-tf-box-trailing-wrapper');
+
+        var tfIcons = document.querySelectorAll('.mdc-textfield__icon');
```

Comment:
`[].slice.call(document.querySelectorAll(...))` so that `forEach` works in IE 11